gadget/ondisk.go

```diff
@@ -164,7 +170,7 @@ var (
 
 // CreateMissing creates the partitions listed in the positioned volume pv
 // that are missing from the existing device layout.
-func (dl *DeviceLayout) CreateMissing(pv *gadget.LaidOutVolume) ([]DeviceStructure, error) {
+func (dl *OnDiskVolume) CreateMissing(pv *LaidOutVolume) ([]OnDiskStructure, error) {
```

Comment:
not a blocker but it feels this belong as a function to gadget/install (which means some other internal things moving to internal or install)

Reply:
Added a note in the code about that.